### ISSUE TITLE
Lock cells that should not be edited

### ISSUE THREAD

**Issue by quantum-dan** [quantum-dan/DeCal#1]

**Comment by quantum-dan** [comment]
Solution doesn't seem to be viable due to serious bugs in Excel's sheet protection.  Upon protecting the sheet and locking the relevant cells, the sheet can't be un-protected (Excel always claims the password is wrong, even when no password is set) and the cells can't be unlocked even from VBA and even when Protect is specifically called with a parameter to still allow VBA to edit locked cells.

In other words, not happening because Excel.
